src/renderer/components/widgets/channels/ChannelInput/ChannelInput.test.tsx

```diff
@@ -1,25 +1,19 @@
 import React from 'react'
 
-import { INPUT_STATE } from '../../../../store/selectors/channel'
-
 import { renderComponent } from '../../../../testUtils/renderComponent'
 import { ChannelInput } from './ChannelInput'
 
 describe('ChannelInput', () => {
   it('renders component input available ', () => {
```

Comment:
This name is a little cryptic. What does "available" mean here?

Reply:
That it renders input component that is not disabled. The name of this test could be "renders component input" as well

Reply:
k